packages/client/eslint.config.js

```diff
@@ -0,0 +1,93 @@
+import js from "@eslint/js"
+import react from "eslint-plugin-react"
+import hooks from "eslint-plugin-react-hooks"
+import simpleImportSort from "eslint-plugin-simple-import-sort"
+import globals from "globals"
+import tseslint from "typescript-eslint"
+
+// Tried to introduce typechecking to this file, but requires
+// * .js to be added to tsconfig.json
+// * // @ts-check at the top of this file
+// * this /** @type {import('@typescript-eslint/utils').TSESLint.FlatConfig.ConfigFile} */
+// ... and even then eslint-plugin-react has no (useful) types .. sigh
+//
+export default [
+  js.configs.recommended,
+  ...tseslint.configs.recommended,
+  {
+    plugins: {
+      // https://github.com/lydell/eslint-plugin-simple-import-sort
+      "simple-import-sort": simpleImportSort,
+      // https://typescript-eslint.io/getting-started/
+      "@typescript-eslint": tseslint.plugin,
+      // https://github.com/jsx-eslint/eslint-plugin-react
+      react,
+      // https://github.com/facebook/react/tree/main/packages/eslint-plugin-react-hooks
+      // compat with ESLint v9, see https://github.com/facebook/react/issues/28313
+      "react-hooks": hooks,
```

Comment:
added hooks plugin .. not FlatConfig-ready, so cannot just add config as `tseslint` above (this is the future model) ... just a few transgressions to fix